chainermn/communicators/__init__.py

```diff
@@ -17,11 +17,13 @@ def create_communicator(
     two_dimensional     OK  Required Each node has multiple NICs or HCAs
     single_node         OK  Required Single node with multiple GPUs
     flat                OK           N/A
+    nccl                OK  Required N/A
     =============== === === ======== =======================================
 
     Args:
         communicator_name: The name of communicator (``naive``, ``flat``,
-          ``hierarchical``, ``two_dimensional``, or ``single_node``)
+          ``hierarchical``, ``two_dimensional``, ``nccl``, or
```

Comment:
I think we do not need hierarchical communicator if we use NCCL 2. But it is still needed because chainermn supports NCCL 1.